### PR TITLE
read modelName from config file

### PR DESCRIPTION
**Pull request by vli11**

### CODE REVIEW

**Review by marcpfuller — APPROVED** [review]

**Review by Brian-McGinn — APPROVED** [review]

File: src/go/go-binding/utilities/util.go [hunk 1]
```diff
@@ -35,3 +41,32 @@ func Clamp(val float32, min float32, max float32) float32 {
...
+		}
+		if strings.Contains(string(line), "\"name\": ") {
+			sections := strings.Split(string(line), ":")
+			names := strings.Split(sections[1], "\"")
+			modelName := names[1]
+			return modelName, nil
+		}
+	}
+	return "", fmt.Errorf("Can not find model name in configfilepath: %v", configfilepath)
+}
```
Comment (jim-wang-yutsung) [inline src/go/go-binding/utilities/util.go]: would suggest parsing config.json use json unmarshal instead of this ad-hoc approach

**Review by jim-wang-yutsung — CHANGES_REQUESTED** [review]

File: src/go/go-binding/utilities/util.go [hunk 2]
```diff
@@ -35,3 +40,40 @@ func Clamp(val float32, min float32, max float32) float32 {
 	}
 	return val
 }
+
+func GetModelNameFromConfig(configfilepath string) ([]string, error) {
```
Comment (jim-wang-yutsung) [inline src/go/go-binding/utilities/util.go]:
```suggestion
func GetModelNameListFromConfig(configfilepath string) ([]string, error) {
```

File: src/go/go-binding/capi.go [hunk 3]
```diff
@@ -460,6 +461,18 @@ func main() {
 	logLevel := OVMS_LogLevelNew(OVMS_LOG_DEBUG)
 	serverSettings.OVMS_ServerSettingsSetLogLevel(logLevel)
 
+	modelNameList, err := utilities.GetModelNameFromConfig(modelConfigPath)
```
Comment (jim-wang-yutsung) [inline src/go/go-binding/capi.go]:
```suggestion
	modelNameList, err := utilities.GetModelNameListFromConfig(modelConfigPath)
```

**Review by jim-wang-yutsung — APPROVED** [review]
LGTM

**Pull request merged**